lib/timeline.js

```diff
@@ -0,0 +1,69 @@
+const guid = require('./utils/guid')
+
+class Timeline {
+  constructor(client) {
+    this.client = client
+  }
+
+  createEventType(applicationId, userId, data, cb) {
+    data['applicationId'] = data['applicationId'] || applicationId
+    return this.client._request(
+      {
+        method: 'POST',
+        path: `/integrations/v1/${applicationId}/timeline/event-types?userId=${userId}`,
+        body: data,
+      },
+      cb,
+    )
+  }
+
+  updateEventType(eventTypeId, applicationId, data, cb) {
```

Comment:
What is `cb`?

Reply:
callback. The acronym appears to be used throughout the codebase so I haven't expanded it here for consistency.